library/network/src/modules/SuSEFirewall.rb

```diff
@@ -2083,6 +2083,86 @@ def GetSupportRoute
       @SETTINGS[:routing]
     end
 
+    def ArePortsOrServicesAllowed(needed_ports, protocol, zone, check_for_aliases)
```

Comment:
This 32 line method can be implemented in 3 lines, if we move the original implementation to the superclass:
```rb
def ArePortsOrServicesAllowed(needed_ports, protocol, zone, check_for_aliases)
   super(needed_port, protocol, zone, false)
end
```